ChangeLog

```diff
@@ -1,3 +1,11 @@
+Version 9.9.4
+-------------
+    * Restructure Purity so that it deletes action results before marking
+	their blobs as live.
+	* Add cache prefix field to purity action results proto
+	* Add cache prefix field to purity blobs proto
+	* Elan List() returns blobs with cache preflix field populated 
+
```

Comment:
nit: This should be version `10.0.0` and the bullet points could be aligned better